src/main/java/appeng/api/storage/ICellCacheRegistry.java

```diff
@@ -0,0 +1,29 @@
+package appeng.api.storage;
+
+import cpw.mods.fml.relauncher.Side;
+import cpw.mods.fml.relauncher.SideOnly;
+
+public interface ICellCacheRegistry {
+
+    @SideOnly(Side.CLIENT)
+    long getTotalBytes();
+
+    @SideOnly(Side.CLIENT)
+    long getFreeBytes();
+
+    @SideOnly(Side.CLIENT)
+    long getUsedBytes();
+
+    @SideOnly(Side.CLIENT)
+    long getTotalTypes();
+
+    @SideOnly(Side.CLIENT)
+    long getFreeTypes();
+
+    @SideOnly(Side.CLIENT)
+    long getUsedTypes();
+
+    @SideOnly(Side.CLIENT)
+    StorageChannel getCellType();
```

Comment:
`@SideOnly(Side.CLIENT)` should not be used by modders unless absolutely necessary. And these methods are called on server.

Reply:
I thought these annotations mean this method just run on client, I'm delete them now.